tests/clds_hash_table_int/clds_hash_table_int.c

```diff
@@ -313,4 +680,530 @@ TEST_FUNCTION(clds_hash_table_delete_after_set_value_succeeds)
     clds_hazard_pointers_destroy(hazard_pointers);
 }
 
+TEST_FUNCTION(clds_hash_table_snapshot_works_with_10000_sequential_key_items)
+{
+    // arrange
+    CLDS_HAZARD_POINTERS_HANDLE hazard_pointers = clds_hazard_pointers_create();
+    CLDS_HAZARD_POINTERS_THREAD_HANDLE hazard_pointers_thread = clds_hazard_pointers_register_thread(hazard_pointers);
+    volatile int64_t sequence_number = 45;
+    CLDS_HASH_TABLE_HANDLE hash_table = clds_hash_table_create(test_compute_hash, test_key_compare, 1, hazard_pointers, &sequence_number, test_skipped_seq_no_cb, (void*)0x5556);
```

Comment:
>CLDS_HASH_TABLE_HANDLE hash_table = clds_hash_table_create(test_compute_hash, test_key_compare, 1, hazard_pointers, &sequence_number, test_skipped_seq_no_cb, (void*)0x5556); [](start = 4, length = 173)

I'd assert on the result these days. There was a time when we did not, but we're doing this now to have a controlled way of failing.